ci/build/build-pipeline.yaml

```diff
@@ -262,6 +262,7 @@ spec:
           trigger: true
 
       - task: run-tests
+        attempts: 2
```

Comment:
Out of curiosity; why are we now attempting the tests twice?

Reply:
Cos they randomly fail sometimes 🤷‍♂

Reply:
As good a reason as any!

Reply:
😬